.github/workflows/dashboard.yml

```diff
@@ -0,0 +1,47 @@
+name: Daily Command
+
+on:
+  schedule:
```

Comment:
I guess it would be useful to run a job also when some branch is merged into master, like:
```yaml
on:
  pull_request:
    branches: 
      - main
    types: [closed]

jobs:
  run-command:
    if: ${{ github.event.pull_request.merged }}
    runs-on: ...
```
@stdevMac Let me know your thoughts

Reply:
Good point! Will include it!